### PR TITLE
Support message thread and thread member APIs

### PR DESCRIPTION
**Pull request by gffking**
Read-only APIs for message threads and thread members.

Also adds the beginnings of a CLI tool for testing the API.

### CODE REVIEW

File: cli/.env-default [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ELATION_BASE_URL=https://app.elationemr.com/api/2.0/
```
Comment (fterrag) [inline cli/.env-default:1]: Can we have the default vars point to the sandbox env?

File: cli/cmd/messaging.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	Run: wrapRunFunc(func(ctx context.Context, client *elation.Client, args []string) error {
+		response, _, err := client.ThreadMemberSvc.Find(ctx, &elation.FindThreadMembersOptions{
+			Patient: findThreadMembersPatients,
+			User:    findThreadMembersUsers,
+		})
+		if err != nil {
+			return err
+		}
+
+		spew.Dump(response)
```
Comment (fterrag) [inline cli/cmd/messaging.go:28]: spew.Dump call

File: cli/go.mod [hunk 3]
```diff
@@ -0,0 +1,23 @@
+module github.com/authorhealth/go-elation/cli
```
Comment (fterrag) [inline cli/go.mod:1]: I don't think this needs to be a separate module

File: cli/cmd/messaging.go (hunk 2 above)
Comment (gffking) [reply cli/cmd/messaging.go:28]: That's on purpose. I don't want to write formatters for every response type.

File: cli/go.mod (hunk 3 above)
Comment (gffking) [reply cli/go.mod:1]: Fair enough. My thinking was by keeping this as a separate module, it avoided making consumers of the client pull in the CLI-specific dependencies like cobra and spew.

File: cli/.env-default (hunk 1 above)
Comment (gffking) [reply cli/.env-default:1]: Sure, can do.

File: cli/go.mod (hunk 3 above)
Comment (fterrag) [reply cli/go.mod:1]: I think it's OK for now - maybe revisit when we feel the need to?

**Review by fterrag — APPROVED** [review]

File: cli/go.mod (hunk 3 above)
Comment (gffking) [reply cli/go.mod:1]: Probably my lack of go-fu, but I'm getting import cycles trying to move this out of the separate package. I'm going to leave this as-is, but if you know a way to resolve this cycles, I'd be happy to refactor.

**Pull request merged**